src/engraving/libmscore/score.cpp

```diff
@@ -2559,6 +2559,11 @@ void Score::removePart(Part* part)
             break;
         }
     }
+    for (auto excerpt : masterScore()->excerpts()) {
+        if (excerpt) {
+            masterScore()->removeExcerpt(excerpt);
```

Comment:
This code just removes all excerpts when removing any part, it doesn't fix the problem

Reply:
I tested with debugger, that ```_excerpt``` after create parts is too empty, so ```if(_excerpt)``` condition is always false and never is executed. I checked in the same place, that adding parts adds axcerpts to MasterScore(). I can search another solution.

Reply:
@RomanPudashkin I put new commit. Is it better? I would ask if searching in this place is a good idea or better search in another place?

Reply:
@RomanPudashkin Yesterday I improved this code. Now removes only excerpts, which contain part, not all. I think, that now is ready to review.

Reply:
No, this solution also won't work. Now, this code will remove all excerpts containing this part, which is not the desired behavior. I don't recommend fixing this bug right now because it is quite complex and requires a deep understanding of how the system works